### PR TITLE
satellite/satellitedb: update node version columns in UpdateCheckIn

### PR DESCRIPTION
**Pull request by cam-a**
What: Update the node's version in overlay cache

Why: We missed the node version columns in the update
 
Please describe the performance impact:

## Code Review Checklist (to be filled out by reviewer)
 - [x] Does the PR describe what changes are being made?
 - [x] Does the PR describe why the changes are being made?
 - [x] Does the code follow [our style guide](https://github.com/storj/docs/blob/master/code/Style.md)?
 - [x] Does the code follow [our testing guide](https://github.com/storj/docs/blob/master/code/Testing.md)?
 - [x] Is the PR appropriately sized? (If it could be broken into smaller PRs it should be)
 - [x] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
 - [x] Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
 - [x] Does any documentation need updating?
 - [x] Do the database access patterns make sense?

### CODE REVIEW

File: satellite/overlay/service_test.go [hunk 1]
```diff
@@ -530,6 +530,10 @@ func TestUpdateCheckIn(t *testing.T) {
 		require.Equal(t, updatedNode.Address.GetAddress(), expectedAddress)
 		require.Equal(t, updatedNode.Reputation.UptimeSuccessCount, actualNode.Reputation.UptimeSuccessCount+1)
 		require.Equal(t, updatedNode.Capacity.GetFreeBandwidth(), int64(12355))
+		require.Equal(t, updatedInfo.Version.GetVersion(), updatedNode.Version.GetVersion())
+		require.Equal(t, updatedInfo.Version.GetCommitHash(), updatedNode.Version.GetCommitHash())
+		require.Equal(t, updatedInfo.Version.GetRelease(), updatedNode.Version.GetRelease())
+		// require.True(t, updatedNode.Version.GetTimestamp().After(info.Version.GetTimestamp()))
```
Comment (stefanbenten) [inline satellite/overlay/service_test.go]: This should be fixed before merging

Comment (cam-a) [reply satellite/overlay/service_test.go]: oops, thanks for catching that

**Comment by stefanbenten** [comment]
@littleskunk You didnt had the label!

**Comment by cam-a** [comment]
That's okay

**Comment by littleskunk** [comment]
@stefanbenten better now? :D

**Pull request merged**